packages/core/admin/admin/src/content-manager/components/DynamicTable/index.js

```diff
@@ -93,24 +68,42 @@ const DynamicTable = ({
           searchable: false,
           sortable: true,
         },
-        cellFormatter(cellData) {
-          const isPublished = cellData.publishedAt;
-          const variant = isPublished ? 'success' : 'secondary';
+        cellFormatter({ publishedAt }) {
+          return <PublicationState isPublished={!!publishedAt} />;
+        },
+      });
+    }
 
-          return (
-            <StyledStatus showBullet={false} variant={variant} size="S">
-              <Typography fontWeight="bold" textColor={`${variant}700`}>
-                {formatMessage({
-                  id: getTrad(`containers.List.${isPublished ? 'published' : 'draft'}`),
-                  defaultMessage: isPublished ? 'Published' : 'Draft',
-                })}
-              </Typography>
-            </StyledStatus>
-          );
+    if (hasReviewWorkflows) {
+      formattedHeaders.push({
+        key: '__strapi_reviewWorkflows_stage_temp_key__',
+        name: 'strapi_reviewWorkflows_stage',
+        fieldSchema: {
+          type: 'custom',
         },
-      },
-    ];
-  }, [runHookWaterfall, displayedHeaders, layout, hasDraftAndPublish, formatMessage]);
+        metadatas: {
+          label: formatMessage({
+            id: getTrad(`containers.ListPage.table-headers.reviewWorkflows.stage`),
+            defaultMessage: 'Review stage',
+          }),
+          searchable: false,
+          sortable: false,
+        },
+        cellFormatter({ strapi_reviewWorkflows_stage }) {
+          return <ReviewWorkflowsStage {...strapi_reviewWorkflows_stage} />;
```

Comment:
Should we be spreading the props here? We don't declare what they are on the component either, imo we should only pass it what it needs to avoid confusion in the future

Reply:
Done in 27753fc03bd24d35a8fc6f3fcd24a43597e5f1c9